src/savegame_json.cpp

```diff
@@ -1370,7 +1370,7 @@ void npc::load( JsonObject &data )
     }
     data.read( "known_to_u", known_to_u );
     data.read( "personality", personality );
-
```

Comment:
It would be nice if you could move `magic` to `Character` here, and move the load and store for it to `Character` as well, rather than have duplicates in `npc` and `avatar`